<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HoursForm_template</class>
 <widget class="QDialog" name="HoursForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="The title of the dialog to set the days of the week">The hours of the day</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="realHoursLabel">
       <property name="text">
        <string>Real hours:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="realHoursListWidget"/>
     </item>
     <item>
      <widget class="QPushButton" name="modifyRealHourPushButton">
       <property name="text">
        <string comment="Modify real hour">Modify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="okPushButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelPushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="hoursLabel">
         <property name="text">
          <string>Hours:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="nHoursSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="hoursListWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="insertHourPushButton">
         <property name="text">
          <string>Insert</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modifyHourPushButton">
         <property name="text">
          <string comment="Modify hour">Modify</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeHourPushButton">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>nHoursSpinBox</tabstop>
  <tabstop>hoursListWidget</tabstop>
  <tabstop>realHoursListWidget</tabstop>
  <tabstop>insertHourPushButton</tabstop>
  <tabstop>modifyHourPushButton</tabstop>
  <tabstop>removeHourPushButton</tabstop>
  <tabstop>modifyRealHourPushButton</tabstop>
  <tabstop>okPushButton</tabstop>
  <tabstop>cancelPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
